### PR TITLE
Feature/serge refactor  docs

### LINKED ISSUE #10: Create Documentation for monorepo
The monorepo architecture is different to the original project. So documentation will be needed to make it clear how to use it.

### PR DESCRIPTION
**Pull request by foxleigh81**
## 🧰 Issue

Closes #10 

## 🚀 Overview: 
Adds a 'development standards' document to assist us with the new ways of working which are about to be implemented. Also updated the issue and pull request templates.

## 🤔 Reason: 
When the new ways of working begin, guidelines will be needed for everyone to adhere to. This provides those guidelines.

## 🔨Work carried out:

- [x] Created V1 of standards document
- [x] Created a new 'hotfix' issue template
- [x] Updated PR template (this is an example of the new template)
- [x] Updated the bug report template to be a little more useful
- [x] Renamed the 'userstory' template to 'feature' and added more details
- [x] Implemented an 'auto assign' bot and added a config for it.

## Confirmations

- [x] I have chosen reviwers for my PR.
- [x] I have assigned myself to this PR.
- [x] I have chosen an appropriate label for the PR.
- [x] I have completed the mandatory sections of this document.
- [x] I have deleted any unused sections.
- [x] I confirm that I have checked for required README updates and acted accordingly.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -7,29 +7,34 @@ assignees: ''
...
 
-**Screenshots**
-If applicable, add screenshots to help explain your problem.
+## 🖥️ Screenshots
+<!-- If applicable, add screenshots to help explain your problem a good animated screenshot can replace 'steps to reproduce'. -->
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
+## 💻 System information
+<!-- Please complete the following information (you can get most of this information from https://whatismybrowser.com) -->
```
Comment (IanMayo) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: I'm impressed by `https://whatismybrowser.com)` - it's very good.

File: .github/ISSUE_TEMPLATE/hotfix.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
```
Comment (IanMayo) [inline .github/ISSUE_TEMPLATE/hotfix.md]: @foxleigh81 - I think we could do with explaining the difference between bug report & hotfix (I don't know...)

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (foxleigh81) [reply .github/ISSUE_TEMPLATE/bug_report.md:28]: It's very good, isn't it! :) There are a few others out there too but I think that's the most user-friendly one.

File: .github/ISSUE_TEMPLATE/hotfix.md (hunk 2 above)
Comment (foxleigh81) [reply .github/ISSUE_TEMPLATE/hotfix.md]: You know what, technically a hotfix is for a release which we don't really have a need for. So I'm going to rethink the bug and hotfix thing a bit.

Comment (foxleigh81) [reply .github/ISSUE_TEMPLATE/hotfix.md]: I've replaced `hotfix` with `amend request` I used hotfix as I'm used to that term but it doesn't technically apply in this context as we don't really have to work on 'released' code we just do work for a next release. 

So now the difference is:

> An amend is different to a bug. A bug is an issue that causes a failure or error somewhere, whereas an amend is something that isn't quite right. For example a page not loading is a bug, a page having a header that needs to be bigger is an amend. 

This information is also in the development standards document :)

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -1,24 +1,46 @@
...
 
 - [x] Tests pass
 
 ## 🖥️ Screenshot
-[If the work is UI related then paste a screenshot of the update here.]
+<!-- [If the work is UI related then paste a screenshot of the update here. Where possible, please use an animated screenshot.] -->
+
+## Confirmations
+
+- [ ] I have chosen reviwers for my PR.
```
Comment (IanMayo) [inline .github/pull_request_template.md]: `reviewers`

File: development-standards.md [hunk 4]
```diff
@@ -0,0 +1,446 @@
...
+
+This section applies to the entire project, regardless of language
+
+#### Coding style
+
+- Use spaces for tabs
+- Use 2 space widths per tab indentation
+- Trim all trailing whitespace
+
+> ❗ For the simplest development process, please ensure youre development environment confirms to the project `.editorconfig` file. There are plugins for most (if not all popular editors), find yours [here](https://editorconfig.org/#download).
```
Comment (IanMayo) [inline development-standards.md]: `your`

File: development-standards.md [hunk 5]
```diff
@@ -0,0 +1,446 @@
...
+
+#### Code comments
+
+Code comments should be written in a formal, imperative style and describe the code or the intended outcome. Please avoid the use of colloquialisms and use present tense.
```
Comment (IanMayo) [inline development-standards.md]: Could you give a couple of examples of good/bad use of `formal, imperative style`?  I'll have to work hard to adopt it, our non-native English speakers may find it even more challenging.

File: development-standards.md [hunk 6]
```diff
@@ -0,0 +1,446 @@
...
+
+##### Unacceptable
+
+```
+mapping-stuff
+```
+
+#### Protected branches
+
+Two protected branches exist, these are `master` and `develop`. `master` is only for releases and will be carried out by @IanMayo, `develop` is the active development branch for our work. All work streams should initially branch of `develop`. Pushing to develop can only be done via a Pull Request (see below).
```
Comment (IanMayo) [inline development-standards.md]: `branch off`

File: development-standards.md [hunk 7]
```diff
@@ -0,0 +1,446 @@
...
+Removes console.log from the 'markerFor' helper on the Mapping component
+```
+
+#### Unacceptable
+
+![A bad example of a commit message](https://i.imgur.com/ajZ8Ff2.png)
+
+### Issues
+
+Ideally an issue will exist for all work. There are two types of issues, a 'bug report' a 'hotfix' and a 'feature'.
```
Comment (IanMayo) [inline development-standards.md]: `three` types

**Review by IanMayo — APPROVED** [review]

**Pull request merged**